### DISCUSSION TITLE
[Basketball] Can't get league info for 2023 league

### DISCUSSION THREAD

**Discussion by chidmuthu** [cwendt94/espn-api#383] [Q&A] [answered]
Creating the League object for a fantasy basketball 2023 categories league fails.

```
Traceback (most recent call last):
  File "basketball.py", line 9, in <module>
    league = League(1468400594, 2023, ESPN_S2, SWID, True)
  File "/usr/local/anaconda3/lib/python3.8/site-packages/espn_api-0.22.1-py3.8.egg/espn_api/basketball/league.py", line 23, in __init__
  File "/usr/local/anaconda3/lib/python3.8/site-packages/espn_api-0.22.1-py3.8.egg/espn_api/basketball/league.py", line 26, in fetch_league
  File "/usr/local/anaconda3/lib/python3.8/site-packages/espn_api-0.22.1-py3.8.egg/espn_api/basketball/league.py", line 31, in _fetch_league
ValueError: min() arg is an empty sequence
```

Same League object with 2022 year instead works fine. Also, debug logging isn't printing anything? Unless it goes to a file somewhere I don't know. Any pointers here would be appreciated!

**Comment by chidmuthu** [comment, marked as answer]
Needed to upgrade to 0.25.0! Thanks for this package, absolutely loving it so far :)

**Reply by cwendt94** [reply]
Glad the newest version worked!

**Reply by srednasykcin** [reply]
Hey guys - just wanted to comment that I'm thrilled something like this is even developed as much as this is! I just started writing python in May 2022 so please excuse any obvious faux pas and naivete.

What needs to be updated to 0.25.0?

**Reply by chidmuthu** [reply]
This python package itself. You should be able to pip upgrade it.

**Reply by srednasykcin** [reply]
Oddly enough I started this project months ago and just picked it back up again. I was running 0.20.0 
ugh

Thanks for clearing this up.

**Comment by JamesPHawk** [comment]
Hey @chidmuthu, I'm running into a very similar issue right now. Is your league set to private?

**Reply by chidmuthu** [reply]
Yeah, but the latest version should still work.
